server/src/main/resources/password/pwm/config/PwmSetting.xml

```diff
@@ -623,6 +623,13 @@
             <value/>
         </default>
     </setting>
+    <setting hidden="false" key="passwordExpirationTimeAttribute" level="2">
+        <ldapPermission actor="proxy" access="write"/>
```

Comment:
Just noted that we don't need write access to this attribute. I'll wait for review in case other changes are needed. Thanks.